### PR TITLE
Rebrand Governance page modals

### LINKED ISSUE #462: Rebrand DAO Dashboard - Governance page modals
Update delegation modals on the Governance page based on the [new designs](https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen/%E2%9F%81--Api3-DAO---Branding-Update?node-id=8676-108208&node-type=frame&m=dev)

Image

Image

Image

### PR DESCRIPTION
**Pull request by Anboias**
Closes #462 

## What does this do?
- updates UI for components listed below
- adds extra modal for Undelegate confirmation

## What's left aside?
There are some inconsistencies in some of the modals regarding the text positioning. Design Team created a [ticket](https://github.com/api3dao/tasks/issues/1535) to adjust them. I suggest we pick it up at a later point, in a separate task, instead of waiting for it

#### Delegate

| Mobile | Desktop |
|--------|--------|
| image | image | 

#### Update delegation

| Mobile | Desktop |
|--------|--------|
| image | image | 

#### Confirm undelegate

| Mobile | Desktop |
|--------|--------|
| image | image | 

#### Vote on Proposal

| Mobile | Desktop |
|--------|--------|
| image | image |

### CODE REVIEW

**Review by peterjurco — COMMENTED** [review]
Nice job 👍

File: src/pages/proposals/forms/choose-delegate-action/choose-delegate-action.tsx [hunk 1]
```diff
@@ -12,17 +13,53 @@ interface Props {
 const ChooseDelegateAction = (props: Props) => {
   const { onUpdateDelegation, onUndelegate, canUpdateDelegation, canUndelegate } = props;
 
+  const [confirmationStep, setConfirmationStep] = useState(false);
+
+  if (confirmationStep) {
```
[the remark this answers is not in the document]
Comment (Anboias) [reply src/pages/proposals/forms/choose-delegate-action/choose-delegate-action.tsx:18]: Not sure why, no. I initially wanted to push back, but then realised it's quite an easy change and it also makes sense to have it.

File: src/components/radio-button/radio-button.module.scss [hunk 2]
```diff
@@ -33,13 +34,32 @@
   }
 }
 
+.radioButtonLabelLarge {
```
[the remark this answers is not in the document]
Comment (Anboias) [reply src/components/radio-button/radio-button.module.scss]: I was going along with what was there already, but I think you're right. Shortening them makes the component more readable

**Review by peterjurco — APPROVED** [review]

File: src/components/input/input.tsx [hunk 3]
```diff
@@ -6,47 +6,50 @@ import styles from './input.module.scss';
 
 type Props = {
   onChange: ChangeEventHandler<HTMLInputElement>;
-  value: string;
-  size?: 'normal' | 'large';
+  value: string | undefined;
```
[the remark this answers is not in the document]
Comment (Anboias) [reply src/components/input/input.tsx:9]: No, it's more of a leftover. Faced some issue during development with the placeholder not showing up, had to do some experiments.
Removed the undefined part

**Review by mcoetzee — APPROVED** [review]

**Pull request merged**